fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleType.java

```diff
@@ -39,6 +39,13 @@ public enum RuleType {
 
     // rewrite rules
     COLUMN_PRUNE_PROJECTION(RuleTypeClass.REWRITE),
+    COLUMN_PRUNE_AGGREGATION(RuleTypeClass.REWRITE),
+    COLUMN_PRUNE_AGGREGATION_CHILD(RuleTypeClass.REWRITE),
```

Comment:
Please unify the naming rules. There appear to be three `xxx`, `xxx_child`, `xxx_children`